Scripts/ParseEmailFiles/ParseEmailFiles.py

```diff
@@ -3307,10 +3308,68 @@ def handle_msg(file_path, file_name, parse_only_headers=False, max_depth=3):
     if not msg:
         raise Exception("Could not parse msg file!")
 
-    email_data = msg.as_dict(max_depth)
+    msg_dict = msg.as_dict(max_depth)
+
+    try:
+        format_string = msg_dict['Headers'].split('Content-type:')[1].split(';')[0]
+    except ValueError:
+        format_string = ''
+    except IndexError:
+        format_string = ''
+
+    headers = []
+    headers_map = dict()  # type: dict
+    header_key = 'initial key'
+    header_value = 'initial header'
+
+    for header in msg_dict['Headers'].split('\n'):
+        if len(header) > 0 and not header == ' ' and 'From nobody' not in header:
```

Comment:
This line seems like it should be in a one-line function to me, so that someone reviewing it could easily understand what it does. It currently requires more know-how than necessary to understand it